src/explore-education-statistics-admin/src/pages/publication/components/ReleaseContributorPermissions.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import releasePermissionService from '@admin/services/releasePermissionService';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import classNames from 'classnames';
+import Button from '@common/components/Button';
+import userService from '@admin/services/userService';
+
+export interface Props {
+  releaseId: string;
+}
+
+const ReleaseContributorPermissions = ({ releaseId }: Props) => {
+  const { value: releaseContributors, isLoading } = useAsyncRetry(
+    () => releasePermissionService.getReleaseContributors(releaseId),
+    [releaseId],
+  );
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (releaseContributors && releaseContributors?.length === 0) {
```

Comment:
Surprised if you need question mark there, given it's already established that it's truthy?

Actually, as with your other component, probably better of writing an additional test at line 23 here, like:

```
if (!releaseContributors) {
    return null;
}
```

Then no need for ? or ! signs.